src/index.html

```diff
@@ -22,6 +22,536 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <!-- header start -->
+
+    <header class="header">
+      <a
+        href="./index.html"
+        class="header__link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo Company Name Logo"
+        />
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+              href="./index.html"
+              class="header__nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="./index.html"
+              class="header__nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="./index.html"
+              class="header__nav-link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="./index.html"
+              class="header__nav-link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="./index.html"
+              class="header__nav-link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="./index.html"
+              class="header__nav-link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="./index.html"
+              class="header__nav-link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="./index.html"
+              class="header__nav-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- header ends  -->
+
+    <!-- cards starts -->
+    <main class="cards">
+      <!-- card #1 -->
+      <section
+        class="product-card"
+        data-qa="card"
+      >
+        <img
+          class="product-card__product-image"
+          src="./images/imac.jpeg"
+          alt="apple-imac-27"
+        />
+
+        <div class="product-card__product-details">
+          <h3 class="product-card__product-details-title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+
+          <p class="product-card__product-details-description">
+            Product code: 195434
+          </p>
+        </div>
+
+        <div class="product-card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star">
+              <div class="stars__star-image"></div>
+            </div>
```

Comment:
This div is redundant, fix it everywhere
```suggestion
            <div class="stars__star"></div>
```